### ISSUE TITLE
A way to Download Backups

### ISSUE THREAD

**Issue by Oiiink** [Minehut/Meta#654]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**
 Java

**Describe the feature**

A way for you to download your saved backup.
**How the feature is useful**

If Minehut ever has problems in the future with their backups I believe there should be a way for the player to download their backups to prevent any data loss. And they should also make a way for them to put their downloaded backup into the server console again.
---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by Reimagining** [comment]
Yes, this would be perfect to secure data better. Especially when unreliable times or bugs arise, such as the migration to 1.17 for most servers.

**Comment by Oiiink** [comment]
> Yes, this would be perfect to secure data better. Especially when unreliable times or bugs arise, such as the migration to 1.17 for most servers.
Especially if you delete a backup to create a brand new server that you think would work well but then after a few seconds of making the idea you realize the idea wasn't as good as you thought and now you lost your backup. Would be pretty sad if that happened.
